src/s3_log_extraction/ip_utils/_index_ips.py

```diff
@@ -6,7 +6,7 @@
 from ..config import get_cache_directory
 
 
-def index_ips(*, seed: int = 0) -> None:
+def index_ips(*, seed: int = None) -> None:
```

Comment:
This was a pretty important bug fix to rerunning extraction (which, with a fixed seed, therefore has a fixed order in which index IPs are drawn and thus forces redraws immediately)